__tests__/Routes/Routes/Users/userRoles.spec.js

```diff
@@ -0,0 +1,38 @@
+const request = require('supertest');
+const server = require('../../api/server');
+
+describe('Should be able to make requests for user data and roles from Auth0.', () => {
+
+    const user = { sub: "github|26471655" }
+    const metaData = {
+        "app_metadata": {
+            "role": "admin"
+        }
+    }
+
+    describe('Should be able to retrieve user data', () => {
+        test('should be able to retrieve full user schema object from Auth0',
+            () => request(server).post(`/api/users/`).send({ "sub": user.sub })
+                .then(result => {
+                    const data = result.body
+                    // console.log('LOGGING', result, data)
```

Comment:
so proud of myself for commenting the whole pr, did it on the og repo. 🤦‍♀  
bold addition of extra space, I'm all about aesthetics.